docs/version0.9.md

```diff
@@ -22,12 +22,116 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# Release notes - version 0.9.0
 
-Version 0.9.0 is currently in the planning phase, as defined in the [release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.9.0/plan).
+# What's new in version 0.9.0
 
+The following new features and notable changes from v.0.8.0 are included in this release:
 
+- [New binaries and supported environments.](#binaries-and-supported-platforms)
+- [The idle tuning feature is now supported on Linux running on Power&reg; Systems and IBM Z&reg; Systems.](#idle-tuning-feature)
+- [A new Garbage Collection (GC) policy is available that performs no housekeeping.](#new-gc-policy)
+- [A command line option is provided to automatically set a larger Java heap size for applications that run in containers.](#modifying-the-default-java-heap-size-for-applications-that-run-in-containers)
+- [You can now specify the maximum Java heap size as a percentage value.](#specifying-the-maximum-java-heap-size-as-a-percentage-value)
+- [The shared classes feature now supports nested jar files.](#shared-classes-support-for-nested-jar-files)
+- [System dump data can now be read to help diagnose problems on Linux platforms.](Direct-Dump-Reader-enabled-on-linux)
```

Comment:
Windows is also supported.

Reply:
Good spot - thanks. Updated here too.

Reply:
The headline was updated, but not the actual topic in the body.

Reply:
Tsk .... lost in the swell before vacation. I will go on holiday eventually! Done